components/toast_wrapper/toast_wrapper.tsx

```diff
@@ -432,7 +425,7 @@ class ToastWrapper extends React.PureComponent {
                 width,
                 onDismiss: this.hideArchiveToast,
                 onClick: this.scrollToLatestMessages,
-                onClickMessage: Utils.localizeMessage('postlist.toast.scrollToBottom', 'Jump to recents'),
+                onClickMessage: localizeMessage('postlist.toast.scrollToBottom', 'Jump to recents'),
```

Comment:
nit: per mattermost convention we would like to keep the Utils. definition so its easier to read where the functions are coming from